packages/@lwc/integration-karma/test/light-dom/scoped-slot/index.spec.js

```diff
@@ -0,0 +1,40 @@
+import { createElement } from 'lwc';
```

Comment:
For the sanity test, I would also recommend adding a simple test for SSR.

Reply:
Added